src/topics/TopicListScreen.js

```diff
@@ -52,7 +59,7 @@ class TopicListScreen extends PureComponent<Props, State> {
   }
 }
 
-export default connectFlowFixMe((state: GlobalState, props) => ({
+export default connect<SelectorProps, _, _>((state, props) => ({
```

Comment:
It'd be a little cleaner if we apply the `<SelectorProps, _, _>` pattern to each of these as we convert from `connectFlowFixMe`. That way, we're maintaining an invariant that that's the new pattern and that's how we always write `connect` calls that have a `props`-taking callback.

Because you've prepared the content of this change already and it's sitting at this later point in your branch, it's possible to quite efficiently move the pieces of it forward to the earlier points where they'd ideally go: in a `git rebase -i`, edit those earlier commits, and do a command like `git checkout master src/topics/TopicListScreen.js` (or replace "master" with whatever branch you're using) to pull forward just one file's changes at a time.

Reply:
Wow, this probably saved me >=20m of work. 🙂

Reply:
Oh, how should I explain the change to `<SelectorProps, _, _>` in all those commit messages?

Reply:
Great :slightly_smiling_face: 

I think it doesn't actually need explanation in the commit messages, because it's the pattern we now universally use for this kind of call.